### DISCUSSION TITLE
How to make a QBtn add chips to a q-select component

### DISCUSSION THREAD

**Discussion by FCnski** [quasarframework/quasar#13929] [General - Components / Directives / etc] [answered]
This might be a pretty dumb question for some of you, but i just can't seem to make this work, i just want to add a button that when pressed will add a chip to a select field instead of having to press the enter key on your keyboard every time, here's where i am at:

```
                       <q-select
                        label="Fontes de Entrada"
                        outlined
                        stack-label
                        v-model="workingTable.fontesEntrada"
                        use-input
                        use-chips
                        multiple
                        new-value-mode="add"
                        hide-dropdown-icon
                        input-debounce="0"
                        :readonly="'RXD'.includes(task)"
                        :rules="[(val) => !!val || 'Campo não pode ser vazio.']"
                        >
                        <template v-slot:append>
                        <q-btn
                          icon="add"
                          @click="(val) => add(val)" />
                        </template>
                      </q-select>
```
i tried different ways too, like using the :after on the q-select component but i am pretty new to this, so i must be doing something very wrong, any help is appreciated! sorry for the bother with the newbie question also!

**Comment by metalsadman** [comment, marked as answer]
check this example in the docs https://quasar.dev/vue-components/select#example--new-value-mode

https://codepen.io/metalsadman/pen/mdxEgpQ

**Reply by FCnski** [reply]
it was way more simpler than i thought, thanks a lot man!
